install_scripts/templates/common/docker-install.sh

```diff
@@ -184,7 +184,7 @@ _installDocker() {
         else
             # Install container-selinux from official source, ignoring errors
             logStep "Installing container-selinux"
-            yum install -y -q container-selinux 2> /dev/null || true
+            yum install -y container-selinux 2> /dev/null || true
```

Comment:
there are 3 more just below this. wanna remove those as well? lines 194, 202, 220